astropy/coordinates/sky_coordinate.py

```diff
@@ -362,22 +362,32 @@ def _parse_inputs(self, args, kwargs):
 
         return valid_kwargs
 
-    def transform_to(self, frame):
-        """
-        Transform this coordinate to a new frame.
+    def transform_to(self, frame, override_defaults=True):
+        """Transform this coordinate to a new frame.
+
+        The frame attributes (e.g., equinox or obstime) for the returned object
+        depend on the corresponding attributes of both the destination ``frame``
+        and the `SkyCoord` instance being transformed, with a precedence that
+        depends on whether ``override_defaults`` is set or not.
+
+        If ``override_defaults`` is `True`:
```

Comment:
See comment below about the keyword arg name, but this line should say that this is the default.  But this gets you to the confusing line of
```
If ``override_defaults`` is ``True`` (which is the default):
```
BTW, shouldn't the single backticks around `True` be double ticks?  Or is that actually a sphinx ref?